tinygrad/tensor.py

```diff
@@ -44,8 +45,7 @@ def __init__(self, data:Union[int, float, list, LazyBuffer, np.ndarray], device=
       assert dtype is None or dtype == data.dtype, "dtype doesn't match, and casting isn't supported"
       lazydata = data if data.device == device else LazyBuffer.loadop(LoadOps.FROM, data.shape, data.dtype, device, src=data)
     elif isinstance(data, np.ndarray):
-      # TODO: create CPUBuffer directly
-      lazydata = LazyBuffer.loadop(LoadOps.FROMCPU, data.shape, dtypes.from_np(data.dtype), device, data)
+      lazydata = LazyBuffer(device, ShapeTracker(tuple(data.shape)), LoadOps, Device[device].buffer.fromCPU(data, **Device.extra_args(device)), dtypes.from_np(data.dtype))
```

Comment:
Create a CPUBuffer then you can copy it later.

Reply:
I see, so it should still defer the load if the device is not the cpu, that can simplify it